x-pack/platform/plugins/shared/fleet/server/routes/schema/errors.ts

```diff
@@ -13,6 +13,7 @@ export const genericErrorResponse = () =>
       statusCode: schema.maybe(schema.number()),
       error: schema.maybe(schema.string()),
       message: schema.string(),
+      attributes: schema.maybe(schema.any()),
```

Comment:
where is `attributes` used?

Reply:
it's used here https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/fleet/server/errors/handlers.ts#L167